SubmittedAssignments/Arti_Kadam/preview.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+    <title>visiting card</title>
+</head>
+<body>
+    <center style="padding-top:40px">
+        <div class="card1-wrapper card-front">
+            <!--card background-->
+            <div class="card1-background">
+                <div class="bottomrect1"></div>
+                <div class="bottomrect2"></div>
+                <div class="bottomrect3"></div>
+            </div>
+
+            <div class="card1-content">
+
+                <div class="org-logo">
+                    <img src="images/cmplogo-removebg-preview.png" class="org-img">
+                </div>
+
+                <div class="org-website">
+                    <div class="org-website-address">www.mmeac.org</div>
+                </div>
+            </div>
+        </div>
+
+        <div class="card2-wrapper card-back">
```

Comment:
**card2-wrapper card-back** not valid
**card-wrapper card-back** should be like this